README_sycl.md

```diff
@@ -114,35 +115,48 @@ Output (example):
 
 2. Build locally:
 
+Note:
+- You can choose between **F16** and **F32** build. F16 is faster for long-prompt inference.
+- By default, it will build for all binary files. It will take more time. To reduce the time, we recommend to build for **example/main** only.
+
+Method using **docker**:
```

Comment:
I suggest adding new chapter for Docker, like
```
## Linux
## Windows
## Docker
```
Because use Docker, the prepare work will be less.
No need to install oneAPI base toolkit.
And docker is highlight in agenda.

Reply:
done in https://github.com/ggerganov/llama.cpp/pull/5228/commits/ddfbc23ca96f4b1afa66ce7df7caf54e1a10c745